sdcm/cluster.py

```diff
@@ -2449,6 +2481,7 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster',
         self.log = SDCMAdapter(LOGGER, extra={'prefix': str(self)})
         self.log.info('Init nodes')
         self.nodes = []
+        self.instance_provision = params.get('instance_provision', default=INSTANCE_PROVISION_ON_DEMAND)
```

Comment:
we set defaults for GCE backend in `defaults/gce_config.yaml`, please remove it  and
add `instance_provision: 'spot'` to the `defaults/gce_config.yaml`

Reply:
I'm in doubt about the removing a code from the base class. I've changed `defaults/gce_config.yaml`, but left `sdcm/cluster.py` unchanged. Please, confirm that I understand you correctly and I need to remove the initialization of `instance_provision` from the base class.

Reply:
no, just remove  default from `params.get('instance_provision', default=INSTANCE_PROVISION_ON_DEMAND)`

Reply:
(our `params` defaults are stored in `defaults` dir)

Reply:
Got it.